test/libdnf/base/test_goal.cpp

```diff
@@ -50,3 +50,46 @@ void BaseGoalTest::test_install() {
     CPPUNIT_ASSERT_EQUAL(0lu, obsoleted_set.size());
 }
 
+void BaseGoalTest::test_install_from_cmdline() {
+    libdnf::Goal goal(base.get());
+    std::filesystem::path repo_path = PROJECT_SOURCE_DIR "/test/libdnf/rpm/repos-data/dnf-ci-fedora/x86_64/wget-1.19.5-5.fc29.x86_64.rpm";
+    auto cmd_pkg = sack->add_cmdline_package(repo_path, false);
```

Comment:
It is not repository path. Better names `package_path`, `pkg_path`, `rpm_path`, ...